src/features/duplicates/components/ConfigureModal.tsx

```diff
@@ -78,18 +78,41 @@ const ConfigureModal: FC<ConfigureModalProps> = ({
             peopleToMerge={peopleToMerge}
           />
         </Box>
-        <Box display="flex" flexDirection="column" width="50%">
+        <Box display="flex" flexDirection="column" marginRight={2} width="50%">
           <FieldSettings
             fieldValues={fieldValues}
             onChange={(field, value) => {
               setOverrides({ ...overrides, [`${field}`]: value });
             }}
           />
+          <Box marginBottom={2} />
+          <Alert severity="info">
+            <AlertTitle>{messages.modal.infoTitle()}</AlertTitle>
+            {messages.modal.infoMessage()}
+          </Alert>
         </Box>
       </Box>
       <DialogActions>
-        <Button variant="text">{messages.modal.cancelButton()}</Button>
-        <Button variant="contained">{messages.modal.mergeButton()}</Button>
+        <Button
+          onClick={() => {
+            setSelectedIds(
+              potentialDuplicate?.duplicates.map((person) => person.id) ?? []
+            );
```

Comment:
What is the purpose of this? Is it to reset so that if the user opens the dialog again it forgets the previous configuration? If that's the case, I think it would be better to put this in a `useEffect()` that depends on the `open` property.